### DISCUSSION TITLE
Consultation on dpgen init_bulk

### DISCUSSION THREAD

**Discussion by 123456Liu123456** [deepmodeling/dpgen#1644] [General]
If the structure of the simulation is simple, can the dpgen init_bulk only be used to generate part of the initial data set? Only deepmd-kit is used for potential function training. And dpgen run is not used for active training. If so, are there any suggestions for a setup that only uses dpgen init_bulk? Thank you.

**Comment by njzjz** [comment]
If `dpgen run` does not generate new data in the first iteration, it means `dpgen init_bulk` is enough.
